exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,34 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(n) {
+if (n < 0){
+  return true;
+}
+return false;
+}
+
+function isBetween5and10(n) {
+  if (n >= 10 && n > 5){
+    return true;
+  }
+  return false; 
+}
+
+
+function isShortName(str) {
+  if (String.length < 7){
+    return true;
+  }
+  return false;
+}
+
+function startsWithD(nam) {
+  if (nam.charAt[0] = "D") {
+    return true;
+  }
+return false; 
+}
 
```

Comment:
good job, just one thing I can add is that in my opinion for returning true is just one if is enough because if it is not true then the function return false.